plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/messages.properties

```diff
@@ -1,10 +1,10 @@
-missing.appengine.xml.element=Cannot find element in appengine-web.xml: 
+missing.appengine.xml.element=Cannot read element in appengine-web.xml:
+cannot.parse.appengine.xml=Cannot parse appengine-web.xml
 task.name.deploy.project=Deploying project
 task.name.stage.project=Staging exploded WAR
 task.name.publish.war=Publish exploded WAR
-project.id.missing=AppEngine project ID is not set, deploy failed. Please set it in the <application> element in WEB-INF/appengine-web.xml
-project.version.missing=AppEngine project version is not set, deploy failed. Please set it in the <version> element in WEB-INF/appengine-web.xml
-deploy.failed.error.message=Deploy failed
+project.id.missing=App Engine project ID is not set, deploy failed. Please set it in the <application> element in WEB-INF/appengine-web.xml
+project.version.missing=App Engine project version is not set, deploy failed. Please set it in the <version> element in WEB-INF/appengine-web.xml
```

Comment:
`App Engine project version is not set, deploy failed. Please set the version in the <version> element in WEB-INF/appengine-web.xml` ?